### PR TITLE
Make statements more accurate

### PR DESCRIPTION
**Pull request by teacup418**
Make statements more accurate

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/the_box_model/index.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ A number of rules dictate when margins do and do not collapse. For further infor
 
 ### Borders
 
-The border is drawn between the margin and the padding of a box. If you are using the standard box model, the size of the border is added to the `width` and `height` of the content box. If you are using the alternative box model then the size of the border makes the content box smaller as it takes up some of that available `width` and `height` of the element box.
+The border is drawn between the margin and the padding of a box. If you are using the standard box model, the size of the border is added to the `width` and `height` of the content box. If you are using the alternative box model then the bigger size of border, the smaller size of content box. As border takes up some of that available `width` and `height` of the element box.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/building_blocks/the_box_model/index.md:246]: To make it grammatically correct :) 

```suggestion
The border is drawn between the margin and the padding of a box. If you are using the standard box model, the size of the border is added to the `width` and `height` of the content box. If you are using the alternative box model, then the bigger the border is , the smaller the content box is, as border takes up some of that available `width` and `height` of the element box.
```

Comment (mind-dance) [reply files/en-us/learn/css/building_blocks/the_box_model/index.md]: wow, thank you ^V^

File: files/en-us/learn/css/building_blocks/the_box_model/index.md [hunk 2]
```diff
@@ -243,7 +243,7 @@ A number of rules dictate when margins do and do not collapse. For further infor
 
 ### Borders
 
-The border is drawn between the margin and the padding of a box. If you are using the standard box model, the size of the border is added to the `width` and `height` of the content box. If you are using the alternative box model then the size of the border makes the content box smaller as it takes up some of that available `width` and `height` of the element box.
+The border is drawn between the margin and the padding of a box. If you are using the standard box model, the size of the border is added to the `width` and `height` of the content box. If you are using the alternative box model, then the bigger the border is , the smaller the content box is, as border takes up some of that available `width` and `height` of the element box.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/building_blocks/the_box_model/index.md:246]:
```suggestion
The border is drawn between the margin and the padding of a box. If you are using the standard box model, the size of the border is added to the `width` and `height` of the content box. If you are using the alternative box model, then the bigger the border is, the smaller the content box is, as border takes up some of that available `width` and `height` of the element box.
```

**Review by Josh-Cena — APPROVED** [review]
I am approving this because it fixes the ambiguous "it" in "it takes up". Otherwise the original statement is accurate too.

**Pull request merged**